vpr/src/device/rr_graph_builder.h

```diff
@@ -240,8 +294,9 @@ class RRGraphBuilder {
      * The main (perhaps only) current use of this metadata is the fasm tool of symbiflow,
      * which needs extra metadata on which programming bits control which switch in order to produce a bitstream.*/
 
-    MetadataStorage<int>& rr_node_metadata_;
-    MetadataStorage<std::tuple<int, int, short>>& rr_edge_metadata_;
```

Comment:
I have deliberately kept them public just to check them once the issue is resolve I will move them to private.